PowerValidatedSolutions.psm1

```diff
@@ -16442,7 +16442,7 @@ Function Export-vROPsJsonSpec {
                                                         'timeSyncMode'                 = "ntp"
                                                         'masterVidmEnabled'            = $true
                                                         'ntp'                          = $jsonInput.ntp
-                                                        'affinityRule'                 = $false
+                                                        'affinityRule'                 = $true
```

Comment:
Won't this place the Cluster Nodes and Proxy Nodes into a single Anti-Affinity rule which is not the desired outcome?

Reply:
Nice!